tools/firely-server-import.rst

```diff
@@ -171,6 +173,23 @@ Supported arguments
 Examples
 --------
 
+Specify a custom settings file **/path/to/your/custom/settings/appsettings.instance.json**.
+
+.. code-block:: bash
+
+  fsi \
+  --settings ./path/to/your/custom/settings/appsettings.instance.json 
+
+.. note::
+  If :code:`--settings` is omitted, FSI searches following folders sequentially and tries to find :code:`appsettings.instance.json`. The first occurrence will be used if FSI finds one, otherwise the default :code:`appsettings.json` will be used.  
```

Comment:
Why do we need this `code` prefix here?

Reply:
No specific reason, just want to highlight these words.
 
Seems that **:code:`...`** has same effect as using **``...``**
I have changed it to **``...``** to make it consistent in this document. :-)